modifiers/string.go

```diff
@@ -89,6 +90,28 @@ func SnakeCase(ctx context.Context, t *mold.Transformer, v reflect.Value, param
 	return nil
 }
 
+// NumCase removes all non-numeric characters. Example: "the price is €30,38" -> "3038". Note: The struct field will remain a string. No type conversion takes place.
+// From https://github.com/leebenson/conform
+func NumCase(_ context.Context, _ *mold.Transformer, v reflect.Value, _ string) error {
+	s, ok := v.Interface().(string)
+	if !ok {
+		return nil
+	}
+	v.SetString(regexp.MustCompile("[^0-9]").ReplaceAllLiteralString(s, ""))
```

Comment:
Can we make the regexes global variables for reuse.

Reply:
I agree. Done.